src/test/java/org/folio/circulationbff/api/CirculationBffRequestsApiTest.java

```diff
@@ -1,29 +1,155 @@
+  private static final String TLR_ALLOWED_SERVICE_POINT_URL = "/tlr/allowed-service-points";
+  private static final String CIRCULATION_ALLOWED_SERVICE_POINT_URL = "/circulation/requests" +
+    "/allowed-service-points";
+
+  @BeforeEach
+  public void beforeEach() {
+    wireMockServer.resetAll();
+  }
+
   @Test
-  void allowedServicePointsReturnsOkStatus() throws Exception {
-    mockMvc.perform(
-        get(buildUrl(ALLOWED_SP_URL_PATH, ALLOWED_SP_QUERY_PARAM_TMP, "create", randomId(),
-          randomId()))
-          .headers(defaultHeaders())
-          .contentType(MediaType.APPLICATION_JSON))
-      .andExpect(status().isOk());
+  void callsModTlrWhenEcsTlrEnabledInCentralTenant() {
+
+    var userTenant = new UserTenant(UUID.randomUUID().toString(), TENANT_ID_CONSORTIUM);
+    userTenant.setCentralTenantId(TENANT_ID_CONSORTIUM);
+    wireMockServer.stubFor(WireMock.get(urlPathEqualTo(USER_TENANTS_URL))
+      .withQueryParam("limit", matching("\\d*"))  // matches any integer or no value
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CONSORTIUM))
+      .willReturn(jsonResponse(asJsonString(new UserTenantCollection().addUserTenantsItem(userTenant)),
+        SC_OK)));
+
+    TlrSettings tlrSettings = new TlrSettings();
+    tlrSettings.setEcsTlrFeatureEnabled(true);
+    wireMockServer.stubFor(WireMock.get(urlMatching(TLR_SETTINGS_URL))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CONSORTIUM))
+      .willReturn(jsonResponse(asJsonString(tlrSettings), SC_OK)));
+
+
+    var operation = "create";
+    var instanceId = UUID.randomUUID();
+    var requestId = UUID.randomUUID();
+    var patronGroupId = UUID.randomUUID();
+
+    var allowedSpResponseConsortium = new AllowedServicePoints();
+    allowedSpResponseConsortium.setHold(Set.of(
+      buildAllowedServicePoint("SP_consortium_1"),
+      buildAllowedServicePoint("SP_consortium_2")));
+    wireMockServer.stubFor(WireMock.get(urlPathEqualTo(TLR_ALLOWED_SERVICE_POINT_URL))
+      .withHeader(HEADER_TENANT, equalTo(TENANT_ID_CONSORTIUM))
+      .willReturn(jsonResponse(asJsonString(allowedSpResponseConsortium), SC_OK)));
+
+    List<StubMapping> stubs = wireMockServer.getStubMappings();
+    // Print all stub mappings
+    for (StubMapping stub : stubs) {
+      System.out.println(stub);
+    }
+
+    doGet(
+      ALLOWED_SERVICE_POINT_PATH + format("?operation=create&requestId=%s&instanceId=%s&patronGroupId=%s",
+        requestId, instanceId, patronGroupId), TENANT_ID_CONSORTIUM)
+      .expectStatus().isEqualTo(200)
+      .expectBody().json("{}");
+
+    wireMockServer.verify(getRequestedFor(urlPathEqualTo(
+      TLR_ALLOWED_SERVICE_POINT_URL))
+      .withQueryParam("requestId", equalTo(requestId.toString()))
+      .withQueryParam("instanceId", equalTo(instanceId.toString()))
+      .withQueryParam("operation", equalTo(operation))
+    );
+  }
+
+  @Test
+  void callsCirculationWhenEcsTlrDisabledOnDataTenant() {
```

Comment:
Could you consider extracting the common code from these tests to avoid duplication ? 
Methods like mockUserTenant, mockSettings, setupAllowedServicePoints, and performRequestAndVerify could be defined to handle the repeated logic